packages/components/package.json

```diff
@@ -20,9 +20,10 @@
     "url": "git+https://github.com/chanzuckerberg/lp-design-system.git"
   },
   "scripts": {
-    "build": "npm run build:clean && npm run build:ts -- --sourceMap false",
+    "build": "npm run build:clean && npm run build:ts && npm run build:types",
     "build:clean": "rm -rf dist/",
-    "build:ts": "tsc -project . --rootDir src/ ",
+    "build:ts": "babel --extensions '.js,.ts,.tsx' --ignore '**/*.stories.tsx','**/*.spec.tsx' --out-dir dist/ src/",
```

Comment:
This is ignoring '**/*.stories.tsx' instead of '**/*.stories.*' because of https://github.com/babel/babel/issues/12008, which is unfortunate.